framework/db/pgsql/QueryBuilder.php

```diff
@@ -206,6 +206,16 @@ public function checkIntegrity($check = true, $schema = '', $table = '')
     }
 
     /**
+     * Builds a SQL statement for truncating a DB table.
```

Comment:
could you add an explanation about this 'consistency' and the exception for PGSQL for future reference?